pkg/server/ca/manager_test.go

```diff
@@ -178,24 +179,25 @@ func (s *ManagerSuite) TestUpstreamSignedWithUpstreamBundle() {
 	})
 	s.initUpstreamSignedManager(upstreamAuthority, true)
 
+	root := upstreamAuthority.UpstreamAuthority.(*fakeupstreamauthority.UpstreamAuthority).Root()
```

Comment:
These changes can all be reverted if you follow the suggestion in fakeupstreamauthority.New.